react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,20 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {PropTypes} from 'prop-types';
 
 import styles from './styles.module.scss';
 
-const Square = () => (
-  <button type="button" className={styles.square}>
-    {/* TODO */}
-  </button>
-);
+function Square(props){
```

Comment:
You can write function Square({ value, onClick }) and use directly without write props